From 11a7816cad28d43de3aea9788ec62b60141b0a56 Mon Sep 17 00:00:00 2001
From: Yonggang Luo <luoyonggang@gmail.com>
Date: Mon, 26 Jan 2026 14:17:29 +0800
Subject: [PATCH] Fixes copy of cygbobcat.dll

cp: 'tmp/libso/cygbobcat.dll' and 'tmp/install//usr/lib/cygbobcat.dll' are the same file
---
 bobcat/icmake/loginstall | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bobcat/icmake/loginstall b/bobcat/icmake/loginstall
index aab6d5fc..477f81af 100644
--- a/bobcat/icmake/loginstall
+++ b/bobcat/icmake/loginstall
@@ -25,7 +25,7 @@ void logInstall(string src, string pattern, string dest)
     entries = findAll("f", src, pattern);
 
     for (idx = listlen(entries); idx--; )
-        run("cp " + src + entries[idx] + " " + dest);
+        run("rm -f " + dest + entries[idx] + "; cp " + src + entries[idx] + " " + dest);
 
     chdir(g_cwd);
     entries = findAll("l", src, pattern);
-- 
2.52.0.windows.1

